frontend/src/components/Board/BuyParts/Bom.js

```diff
@@ -10,6 +11,7 @@ const Bom = ({ length, parts, tsv }) => {
   const [collapsed, setCollapsed] = useState(true)
 
   useEffect(() => {
+    // eslint-disable-next-line no-shadow
     const diff = length - 7
     setDiff(diff)
```

Comment:
What's being shadowed? I don't think we should make exceptions for this rule, we should either rename variables or decide we don't want this rule.